tests/test_slurm.py

```diff
@@ -112,24 +112,45 @@ class TestSbatchTemplate(unittest.TestCase):
     SBATCH_PRELUDE = textwrap.dedent(
         """\
         #!/bin/bash
-        {sbatch_args}
-        module load nix/spack
-        spack load nix
+        {sbatch_args}{spack_prelude}
     """
     )
 
-    def _test_template(self, yaml_file, uc, expected_sbatch_opts, assert_method=None):
+    SPACK_PRELUDE = textwrap.dedent(
```

Comment:
what is that for?

Reply:
Other tests of this class uses a custom sbatch template writing this prelude